sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -351,6 +353,16 @@ def _make_metric_list_to_collect(self, custom_metrics):
                 cfg = {'name': name, 'table': table, 'column': column, 'instance_name': db_name, 'tags': tags}
                 metrics_to_collect.append(self.typed_metric(cfg_inst=cfg, table=table, column=column))
 
+        # Load database files
+        # TODO: if self.dbm_enabled
```

Comment:
are you planning to hide this behind the dbm flag?

Reply:
Yup, will update later.

Reply:
Scratch this. I decided there was no strong argument to gate it behind DBM. So I decided to remove the TODO.